tour/data-model.md

```diff
@@ -17,12 +17,12 @@ The **value** is where the actual data is stored. For brevity, we often refer to
 Take a closer look at the Mutation object created in the first exercise:
 ```java
 Mutation mutation = new Mutation("id0001");
-mutation.put("hero","alias", "Batman");
+mutation.at().family("hero").qualifier("alias").put("Batman");
```

Comment:
I don't know "at" is the right verb here. Perhaps "of" seems more sensible? It's a mutation "of" a family/qualifier, not "at" a family/qualifier.

Reply:
I would create an issue for that.  I am just using the current API.

Reply:
The tour, and our other example code allow us to exercise and question the usability of our API. The question here is whether "at" is grammatically appropriate, in the context of this specific example. If yes, then I won't do anything. If no, then I'll create a follow-on ticket.

Reply:
I think the the issue where this name was discussed `at` was used thinking that the family and qualifier are like coordinates/positions/locations within a row.  It seems like there was a lot of discussion on the issue about this name.  Personally I like the name.

Reply:
Thanks. I agree `at` makes more sense with `put`. `of` might make more sense with `set`, but I'm fine with how it is.